recipe/meta.yaml

```diff
@@ -108,13 +114,11 @@ requirements:
     - cffi
     - future
     - libuv                           # [win]
-    # The test suite uses features only available in numpy 1.17+. We override
-    # numpy pinnings here due to a bug in conda-build mishandling zip key
-    # overrides in local cbc.yaml files. Remove and/or adjust this on the next
-    # upgrade.
-    - numpy >=1.17                    # [not (osx and arm64) and py<310 ]
-    - numpy                           # [    (osx and arm64) or  py>=310]
-    - pip
+    # Pinnings as per pytorch/.circleci/docker/common/install_conda.sh,
+    # except for python 3.8 because we don't have the numpy version required
+    - numpy =1.21.2 # [py>=310]
```

Comment:
IMHO we should pin here without assignment.  for py>=310 version '>=1.21.2,<1.22.0' should be better

Reply:
Done